src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+
+    public static Task[] tasks = new Task[100];
+    public static int numTasks = 0;
+
+    public static void listTasks(){
```

Comment:
I think there should be a space between the closing bracket and open brace. You can use the reformat code option for the IDE to do it for you